commands/list/nodepools/command.go

```diff
@@ -296,6 +318,74 @@ func getOutput(nps []*models.V5GetNodePoolsResponseItems, outputFormat string) (
 	return columnize.Format(table, colConfig), nil
 }
 
+func getOutputAzure(nps []*models.V5GetNodePoolsResponseItems) (string, error) {
+	azureInfo, err := nodespec.NewAzureProvider()
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	headers := []string{
+		color.CyanString("ID"),
+		color.CyanString("NAME"),
+		color.CyanString("AZ"),
+		color.CyanString("VM SIZE"),
+		// color.CyanString("NODES MIN/MAX"),
```

Comment:
Will this be used or not? We shouldn't merge it commented out.

Reply:
We (Celestial) had some discussions about the topic and autoscaling is going be implemented probably after the node pools, so I'll remove this for now.